lib/tasks/price_policies.rake

```diff
@@ -18,4 +18,27 @@ namespace :price_policies do
     end
   end
 
+  # bundle exec rake 'price_policies:update_usage_rate'
+  # bundle exec rake 'price_policies:update_usage_rate[commit]'
+  desc "Update current price policies that have usage_rate"
+  task :update_usage_rate, [:commit] => :environment do |_t, args|
+    commit = args[:commit].to_s == "commit"
+
+    policies = PricePolicy.current.where.not(usage_rate: nil)
+
+    puts "Updating #{policies.count} price rules ..."
+    policies.each do |policy|
+      if policy.usage_rate.nil? || policy.usage_subsidy.nil?
+        puts "Skipped #{policy.id}"
+      else
+        puts "#{policy.id} Before: #{policy.usage_rate.to_f}, #{policy.usage_subsidy.to_f}"
+        policy.usage_rate = policy.hourly_usage_rate.to_f.round(2)
+        policy.usage_subsidy = policy.hourly_usage_subsidy.to_f.round(2)
+        policy.save if commit
```

Comment:
if this fails, it will return `false`, would you rather use `save!` so that it is obvious that it failed? or check the return code and `puts` if it fails? (so that you don't stop the update for others)